youtube_dl/extractor/porntrex.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    urlencode_postdata,
+    ExtractorError,
+)
+
+
+class PornTrexIE(InfoExtractor):
+    '''Class for downloading Porntrex video.'''
+    _NETRC_MACHINE = 'porntrex'
+    _VALID_URL = r'https?://(?:www\.)?porntrex\.com/video/(?P<id>[0-9]+)/.*'
+    _TEST = {
+        'url': 'https://www.porntrex.com/video/519351/\
+                be-ariyana-adin-breaking-and-entering-this-pussy',
```

Comment:
Do not split literals.

Reply:
Done.
The length of the lines made pylint to complain - but resolved now.